HULKs/hulk
# arms up for goalkeeper during set for penalty kicks

## Why? What?

The arms up part of `ArmsUpSquat` either takes a long time or is unstable. We can take advantage of the permission to move our arms during set in order to move the arms up then and only have to sit down during the actual playing phase, increasing speed and stability.
Additionally or current `ArmsUpSquat` technically counts as `WideStance` and results in a standard removal penalty after 10 seconds.
The new ArmsUpStand and ArmsUpSquat pull the arms in closer and are separate motions.

Fixes #1164 

## ToDo / Known Issues

Depends on #1161 so that the motion can be properly adjusted for the wide stance

## Ideas for Next Iterations (Not This PR)

## How to Test

For both the penalty kick and penalty shootout the goalkeeper should now raise its arms when the game state transitions into set.
For the penalty kick this happens from ready to set, during the penalty shootout this happens when the player is selected as the goalkeeper.
When the whistle is blown the robot should now only need to sit down
